tests/client/client.rs

```diff
@@ -0,0 +1,21 @@
+use bson::Bson;
+use mongodb::client::MongoClient;
+
+#[test]
+fn list_databases() {
```

Comment:
`list_databases()` removed in favor of `database_names()`, which returns the array of db names. `list_collections()` still uses a cursor (assuming 3.0+ for now).